src/configValidator/index.test.js

```diff
@@ -1,38 +1,73 @@
 /* eslint no-console:0 */
 import test from 'ava'
 import sinon from 'sinon'
+
 import configValidator from './index'
 
-let originalConsole
+const EMPTY_RESULT = {
+  errors: [],
+  warnings: [],
+  warningMessage: undefined,
+  errorMessage: undefined,
+}
 
-test('no log with no error or messages', t => {
-  setup()
-  configValidator('Webpack Config', {}, [])
-  t.false(console.log.called)
-  cleanUp()
+test('No errors or warnings result in an return object with sensible "empty" values', t => {
+  const result = configValidator('Webpack Config', {}, [])
+  t.same(result, EMPTY_RESULT)
 })
 
-test('logs error messages', t => {
-  setup()
+test('Errors are returned in the `errors` property of the return object ' +
```

Comment:
Eslint `max-len` = 120 characters would yell at me. `eslint-disable-line` for `max-len` seemed awkward to me. ;)